blob/driver/driver.go

```diff
@@ -213,12 +222,12 @@ type Bucket interface {
 	// opts is guaranteed to be non-nil.
 	ListPaged(ctx context.Context, opts *ListOptions) (*ListPage, error)
 
-	// NewRangeReader returns a Reader that reads part of an object, reading at
-	// most length bytes starting at the given offset. If length is negative, it
+	// NewReader returns a Reader that reads an object, reading at
+	// most opts.Length bytes starting at opts.Offset. If opts.Length is 0, it
```

Comment:
Should we add that implementation can expect both offset and length be >= 0?

Reply:
Done (in `driver.ReaderOptions`).